nuxeo-jsf/nuxeo-platform-contentview-jsf/src/main/java/org/nuxeo/ecm/platform/contentview/json/DocumentModelToJSON.java

```diff
@@ -49,12 +48,8 @@
 
     Log log = LogFactory.getLog(DocumentModelToJSON.class);
 
-    protected static final DateFormat dateFormat;
-
-    static {
-        dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZZ");
-        dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
-    }
+    protected static final FastDateFormat dateFormat = FastDateFormat.getInstance("yyyy-MM-dd'T'HH:mm:ssZ",
```

Comment:
FastDateFormat, not something I have heard of.  Should we be using it elsewhere?

Reply:
We are, see https://jira.nuxeo.com/browse/NXP-22091